scripts/launch.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 
+export HOSTNAME_IP_ADDRESS=$(hostname -i)
+
 CONFIG_FILE=${QDROUTERD_HOME}/etc/qdrouterd.conf
+CONFIG_MAP_FILE=/etc/qpid-dispatch/qdrouterd.conf.template
+
```

Comment:
There is an env var QDROUTERD_CONF that can be set to a file (or multiple files) and will then be used as a template: https://github.com/interconnectedcloud/qdrouterd-image/blob/master/scripts/configure.sh#L17